README.md

```diff
@@ -50,8 +50,7 @@ This list contains only stage 1 proposals and higher that have not yet been with
 ### Stage 1
 
 | :rocket: | Proposal                                                                                     | Author                                               | Champion                                             |
-| -------- | -------------------------------------------------------------------------------------------- | ---------------------------------------------------- | ---------------------------------------------------- |
-|          | [`Date.parse` fallback semantics][date-parse]                                                | Morgan Phillips                                      | Morgan Phillips                                      |
+| -------- | -------------------------------------------------------------------------------------------- | ---------------------------------------------------- | ---------------------------------------------------- |                                    |
```

Comment:
This line shouldn't be changed. I can't see exactly what the change was, but it broke the table formatting, at least in the `View` preview in the PR.